global/scripts/MyelinMap_BC.sh

```diff
@@ -185,6 +187,9 @@ ${Caret7_Command} -cifti-math "Var - Bias" ${NativeBCMapToUse} \
 	-var Bias ${NativeBiasField}
 
 log_Msg "_BC Myelin map in the native mesh space: ${NativeBCMapToUse}"
-# TODO: add gifti generation according to one argument
-# -cifti-separate-all
+# gifti generation in native mesh space (BiasField and _BC files)
+if ((GiftiOutputBool)); then
+	${Caret7_Command} -cifti-separate-all ${NativeBiasField} -left ${NativeFolder}/${Subject}.L.BiasField${RegNameInOutputName}.native.dscalar.nii -right ${NativeFolder}/${Subject}.R.BiasField${RegNameInOutputName}.native.dscalar.nii
```

Comment:
the file extensions are not correct.

Reply:
This (and its option) should probably be removed.

Reply:
apparently